wizards/PolicyAutomation/PolicyAutomationWizard.tsx

```diff
@@ -220,6 +226,19 @@ export function PolicyAutomationWizard(props: {
                             }
                         }}
                     />
+                    <WizNumberInput
+                        hidden={(item) => item.spec?.mode !== 'everyEvent'}
+                        path="spec.delayAfterRunSeconds"
+                        label="DelayAfterRunSeconds"
+                        labelHelp="DelayAfterRunSeconds is the minimum seconds before an automation 
+                        can be retriggered on the same cluster.                                 
+                        When a policy is violated, the automation runs one time in the delay period, 
+                        after which it runs every time for the new violation. 
+                        If the policy is violated multiple times during the delay period and kept violated, 
+                        the automation runs one time in the delay period and another time after it.
+                        The default is 0 seconds and only be applicable for the everyEvent mode."
+                        helperText="The period in seconds."
```

Comment:
DelayAfterRunSeconds is the minimum seconds before an automation can be restarted on the same cluster. When a policy is violated, the automation runs one time in the delay period, after which it runs every time for a new violation. If the policy is violated multiple times during the delay period and kept in the violated state, the automation runs one time in the delay period and another time after it. The default is 0 seconds and is only applicable for the everyEvent mode.